workflow.txt

```diff
@@ -1,10 +1,9 @@
+fork on github
 git clone
 git checkout -b my-solution
-git commit -m ""
-git fetch --all
 git add .
-git pull upstream master
-git merge my-solution
-fork on github
+git commit -m ""
 git checkout master
+git merge my-solution
 git push origin my-solution
+git pull upstream master
```

Comment:
👍  typically on hw submission only need:

```
fork on github
git clone
git checkout -b my-solution
git add .
git commit -m ""
git push origin my-solution
```
Then you can submit a PR from there...